pkg/operation/botanist/controlplane.go

```diff
@@ -116,6 +116,54 @@ func (b *Botanist) deleteNamespace(ctx context.Context, name string) error {
 	return err
 }
 
+// EnsureClusterIdentity ensures that Shoot cluster-identity ConfigMap exists and stores its data in the operation
```

Comment:
Can you add to the description that this updates the Shoot.Status in the garden cluster as well as the identity-config map in the seed cluster?